src/main/resources/stashpullrequestbuilder/stashpullrequestbuilder/StashBuildTrigger/config.jelly

```diff
@@ -31,6 +31,9 @@
     <f:entry title="Build only if PR is mergeable" field="checkMergeable">
       <f:checkbox default="false"/>
     </f:entry>
+    <f:entry title="Probe the Stash REST API for merge status just so it recalculates the refspec" field="checkProbeMergeStatus">
```

Comment:
The title is very long compared to other titles. It would be better to shorten the name and move the explanation to the help file. This plugin only has help for two fields, see files `help-buildFailedComment.html` and `help-buildFailedComment.html`. The way to add help is described at https://wiki.jenkins.io/display/JENKINS/Basic+guide+to+Jelly+usage+in+Jenkins#BasicguidetoJellyusageinJenkins-Helpfiles
Suggested title: `Probe Stash for merge status`
The help should probably have details about the race condition,

Reply:
![config-screenshot](https://user-images.githubusercontent.com/1317472/52191081-afb83080-27f7-11e9-8129-ae014d3caf77.png)

Reply:
Thanks, I thought of that too, as I got acquainted with Jelly+Help usage by example in another plugin :)

Reply:
Posted the commit.